### PR TITLE
Fix broken deployment on node:17-alpine

### PR DESCRIPTION
**Pull request by danielduong8**
Use specific version 16.13-alpine instead of latest for Docker images

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # => Build client
-FROM node:alpine as client_builder
+FROM node:16-alpine as client_builder
```
Comment (andy9775) [inline Dockerfile:2]: Did `17` break things or `17.0.1`?

Comment (plausibly) [reply Dockerfile:2]: When I tested this, node 17 was broken it gave some hash error. However, the other images weren't running due to a bash error:

Did you fix the `/bin/sh envsubst not found` issue? I was trying node images and was getting that error.

Comment (danielduong8) [reply Dockerfile:2]: I think 17 and 17.0.1 are identical according to Docker Hub.

The following tags are also all equivalent: 17, 17-bullseye, 17.0, 17.0-bullseye, 17.0.1, 17.0.1-bullseye, bullseye, current, current-bullseye, latest.

Comment (plausibly) [reply Dockerfile:2]: Deploy it on heroku and see if it works

Comment (danielduong8) [reply Dockerfile:2]: Works

Comment (andy9775) [reply Dockerfile:2]: ya, so 17 and latest all evaluate to 17.0.1. If 17.0.0 is fine, lets use that version.

Comment (danielduong8) [reply Dockerfile:2]: I think 17.0.0 is also broken.

Comment (plausibly) [reply Dockerfile:2]: The issue is with version 17 and above, so we need to use 16. https://github.com/webpack/webpack/issues/14532

**Review by plausibly — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (andy9775) [reply Dockerfile:2]: Sounds good!

**Review by andy9775 — APPROVED** [review]
lgtm

**Pull request merged**